### ISSUE TITLE
No "Recharge Duration" option

### ISSUE THREAD

**Issue by Sedohr** [zinic/crawl#17]
I could have sworn in our first iteration of the Crawl we had access to an Aspect Rule called "Recharge" or something similar. In this scenario it worked similar to "Delay" in costs, but instead of it being delayed it would take time after use before you could use it again. Thinking it that way though, it might need to refund less AP since you get more "bang for your buck" this way. As your ability is usable instantly once and then it goes on recharge, compared to the "Delay" where you literally cannot use said aspect until AFTER the delay (which in most scenarios is worse than a recharge since you must wait to use it).

tl;dr requesting a "Recharge Duration" Aspect Rule that can be tagged, indicating that after an aspect/item is used it must cooldown before you can use it once more.
